.github/workflows/on-osx-latest.yml

```diff
@@ -0,0 +1,32 @@
+name: on-osx-latest
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+#      - name: Install GitHub CLI
+#        run: |
+#          brew update
+#          brew install gh
```

Comment:
Why could we potentially need GitHub CLI?

Reply:
That's why I left it as a comment. But I don't know for now. Views?

Reply:
As far as I know there is (currently) no need for GitHub CLI to be installed on the CI. As long as it is not actually required I would prefer to remove the comments to avoid any confusion.